src/components/StoryCreationComponent.js

```diff
@@ -68,78 +52,122 @@ class StoryCreationComponent extends Component {
   };
 
   handleChangeTags = event => {
-    console.log(this.state.tagsField);
     let tag = event.target.value.toLowerCase();
     tag = tag.replace('\\', '');
     tag = tag.replace('*', '');
+    if (_.endsWith(tag, ' ') && tag.length == 1) {
+      this.setState({
+        tagsInputValue: ''
+      });
+      document.getElementById('tags-input-field').value = '';
+      return;
+    }
     if (_.endsWith(tag, ' ')) {
       let newArr = this.state.tagsField;
```

Comment:
This syntax won't create new array, the `newArr` variable is still pointing to the same memory addr as `this.state.tagField`. Refer to my comment on line 62 for changes

Reply:
done.

Reply:
Done.